ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -344,6 +348,8 @@ class SecondaryNavBar extends React.PureComponent {
               <span
                 className="btn btn-view-nav btn-sm"
                 onClick={toggleFieldFilterVisible}
+                role="button"
+                tabIndex="0"
```

Comment:
In many cases, it's preferable to use elements with semantic meaning instead of spans and div's. You can use the components from the react strap library instead.